### PR TITLE
Open link under cursor using Alt-O

### PR DESCRIPTION
**Pull request by ntdkhiem**
https://github.com/BoraxTheClean/adaptable-antelopes/issues/61

### CODE REVIEW

**Review by SharpBit — APPROVED** [review]

File: src/navigation/menu_bar.py [hunk 1]
```diff
@@ -378,4 +372,9 @@ def convert_to_emoji(event: KeyPressEvent) -> None:
             """Convert text to emoji using Ctrl-E"""
             self.do_convert_to_emoji()
 
+        @bindings.add("escape", "o")
```
Comment (BoraxTheClean) [inline src/navigation/menu_bar.py:568]: This is escape-o on my mac. FYI, I know alt keys can be weird.

**Review by BoraxTheClean — CHANGES_REQUESTED** [review]
Nice work!

Can we add this feature to the docs (perhaps after merging https://github.com/BoraxTheClean/adaptable-antelopes/pull/73) and add an example to the welcome page?

**Comment by ntdkhiem** [comment]
> Nice work!
> 
> 
> 
> Can we add this feature to the docs (perhaps after merging https://github.com/BoraxTheClean/adaptable-antelopes/pull/73) and add an example to the welcome page?

Sure!

**Review by ghost — APPROVED** [review]
Good work.

**Review by BoraxTheClean — APPROVED** [review]

**Pull request merged**